index.php

```diff
@@ -10,65 +10,54 @@
 
 class DesignExperiments {
 
+	/**
+	 * List of all CSS files
+	 */
+	private $design_experiment_css_files;
+
+	private $meta_data = array();
+
 	function __construct() {
-		
+
 		// Generate a list of all CSS files
-		$this->design_experiment_css_files = glob( 
-			plugin_dir_path( __FILE__ ) . 'css/*.css'
-		);
-		
-		$this->meta_data =
-			 $this->get_design_experiment_meta_data();
+		$this->design_experiment_css_files = glob( plugin_dir_path( __FILE__ ) . 'css/*.css' );
+
+		$this->get_design_experiment_meta_data();
 
 		// Add admin actions
 		add_action( 'admin_menu', array( $this, 'design_experiments_add_settings_page' ) );
 		add_action( 'admin_init', array( $this, 'design_experiments_settings' ) );
-		add_action( 'admin_enqueue_scripts', array( $this, 'design_experiments_enqueue_stylesheets' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'design_experiments_enqueue_stylesheets' ), 100 );
 	}
 
-
 	/**
 	* Gets the meta data from each design experiment.
 	* return array $meta_data each experiment with its meta data.
 	*/
 	private function get_design_experiment_meta_data() {
-		$meta_data = [];
-		foreach ($this->design_experiment_css_files as $key => $file) {
-			preg_match(
-				'/^\/\*(\{.*?\})\*\//ism',
-				file_get_contents( 
-					$file 
-				),
-				$data
-			);
-			if ( ! empty( $data[1] ) ) {
-				$meta_data[basename($file, '.css')] = json_decode($data[1]);
+		foreach ( $this->design_experiment_css_files as $file ) {
+			$file_content = file_get_contents( $file );
```

Comment:
We should probably use WP's `get_file_data()` here. The JSON format is quite fragile when editing by hand :) 

That will change the `$meta` to be an associative array (can cast to an object if necessary), and also the stylesheets headers will need to be like the current theme headers, instead of:
```
/*{
	"title": "Default Experiment",
	"details": "The default plugin stylesheet.",
	"pr": "https://github.com/WordPress/design-experiments/"
}*/
```
simply:
```
/*
	title: Default Experiment
	details: The default plugin stylesheet.
	pr: https://github.com/WordPress/design-experiments/
*/
```

Reply:
Hmm, I should probably make a PR :)